src/libraries/System.Linq.Queryable/src/System/Linq/EnumerableQuery.cs

```diff
@@ -53,6 +54,7 @@ public EnumerableQuery(Expression expression)
 
         Type IQueryable.ElementType => typeof(T);
 
+        [UnsupportedOSPlatform("browser")]
```

Comment:
What makes these methods unsupported on browser?

Reply:
hm yeah this sounds like a false-positive.